main.py

```diff
@@ -139,9 +139,9 @@ def calc_election_results(widget):
         candidate_sum = public_key.encrypt(0)
         try:
             # (voter_id, encrypted_vote, signature, c_id)
-            query = ("SELECT ctxt FROM votes WHERE c_id = " + str(candidate_list[i][2]))
-            cursor = cnx.cursor()
-            cursor.execute(query)
+            cursor = cnx.cursor(prepared=True)
+            query = ("SELECT ctxt FROM votes WHERE c_id = %s")
+            cursor.execute(query, (str(candidate_list[i][2]),))
```

Comment:
Should there be a comma a the end of the list in the second argument? It is present at the end of the lists in the reference material so I included them.